### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by fabrixxm**
Please check for completeness and grammar

### CODE REVIEW

**Comment by n4rky** [comment]
Quoting fabrixxm <notifications@github.com>:

> Please check for completeness and grammar

On line 10, should the first word be 'Fix?'

On line 30, you refer to "the new nodeinfo protocol (predecessor of  
current statistics.json)." Would a new protocol not instead be a  
successor?


-- 
David Benfell <benfell@parts-unknown.org>

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+	The vagrant VM usage was changed so that the "installation" is now done automatically on the first run. Example users are automatically put into the database (silke)
+	Buttons to insert images or attachment to a post use a popup browser to select a previously uploaded item or upload a new one (fabrixxm, rabuzarus)
+	Improvements in contact handling (annando)
+	Friendica node can now query other nodes about their users and the contact lists (annando)
+	Contact recommendation is done only for recently active users (annando)
+	Admins can opt for search the local DB for contacts instead of the global directory (annando)
+	The global directory is queried in the background to update local DB and improve similar searches in the future. (annando)
+	By communication over the Diaspora protocol, red#matrix sources are now correctly identified, hubzilla is detected (annando)
+	Adopt limitation of usage of "-" in username to avoid conflicts with GNU Social and Diaspora (annando)
+	Option to automatically follow OStatus contacts was moved from addon to the core (annando)
```
Comment (annando) [inline CHANGELOG]: This is duplicated in the text.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+	The [url] tag now also suppots ftp, mailto, gopher links (annando)
+	An "inspect queue" module was added to the admin panel (tobiasd)
+	Fix some missing SQL data escapes (fabrixxm)
+	Improved the accessability of the web UI for better screen reader compatibility (annando)
```
Comment (annando) [inline CHANGELOG]: "accessibility" is the right word.

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+	Added access keys (annando)
+	Support for the public relay server of Diaspora (annando)
+	Support for the new nodeinfo protocol (successor of current statistics.json), addon deprecated as functionality has been moved in core (annando)
```
Comment (annando) [inline CHANGELOG]: "into the core"

**Pull request merged**